contract_operating_unit/views/contract_view.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!--FORM view-->
+    <record id="contract_contract_form_view" model="ir.ui.view">
+        <field name="name">contract.contract form view (in contract_payment_mode)</field>
+        <field name="model">contract.contract</field>
+        <field name="inherit_id" ref="contract.contract_contract_form_view" />
```

Comment:
```suggestion
        <field name="inherit_id" ref="contract.contract.contract_contract_customer_form_view" />
```
Otherwise, OU in supplier invoice will has 2 field,

![image](https://user-images.githubusercontent.com/1973598/119930509-c4f99000-bfa9-11eb-907b-939f3c5d7a79.png)